cli/tests/testdata/f1.yaml

```diff
@@ -23,7 +23,7 @@ timescaledb-single:
 #   https://github.com/timescale/promscale/tree/master/helm-chart
 promscale:
   enabled: true
-  image: timescale/promscale:latest
+  image: vineeth97/promscale:backfill-latest
```

Comment:
This file is being used for ha tests. So needs a change once promscale `0.4.0` is out.